src/Kaleidoscope-Ranges.h

```diff
@@ -49,6 +49,8 @@ enum : uint16_t {
   SC_LAST,
   REDIAL,
   TURBO,
+  DYNAMIC_MACRO_FIRST,
+  DYNAMIC_MACRO_LAST = DYNAMIC_MACRO_FIRST + 31,
```

Comment:
I would probably break that magic number out into its own constant defined somewhere else, possibly over in the dynamic macros code? At the very least, just broken out into its definition, just to make it clear that the number is "how many dynamic macros we get"

Reply:
Can we do that in a separate PR, where I lift all the other magic numbers in Ranges out? (There's... many of them)

Reply:
Yeah. That's ok